src/org/labkey/test/util/exp/SampleTypeAPIHelper.java

```diff
@@ -177,4 +179,30 @@ public static Map<String, Integer> getRowIdsForSamples(String containerPath, Str
         return rowIds;
     }
 
+    /**
+     * Get sample state IDs defined in the specified folder. Useful for updating sample status via API
+     * @param containerPath Path to the folder where the sample statuses are defined
```

Comment:
Sample state is values like "Available", "Consumed", "Locked", etc..?
Those are application only. In a LKSM the query browser shows me values in core/DataStates in the project folder only, subfolders had not data.
Maybe it's different in a non App project?